shared/src/common/enums/exceptions/custom-exception-desc.enum.ts

```diff
@@ -0,0 +1,5 @@
+enum CustomExceptionDesc {
+  BAD_CREDENTIALS = 'Bad credentials',
+}
+
```

Comment:
```suggestion
```
we should create a `ValidationMessage` enum. And use it. Create it on side application where you need it